R/alt_aggregate_counts.R

```diff
@@ -66,7 +65,7 @@ alt_aggregate_counts <- function(
         state_df <- fac_long_df %>%
             mutate(Date = lubridate::floor_date(Date, round_)) %>%
             rename(UCLA = value) %>%
-            # filter(!is.na(UCLA)) %>%
+            filter(!is.na(UCLA)) %>%
```

Comment:
I don't know why this line (and the same one below) was commented out – but I don't think we want it to be! This caused us to drop non-NA facility-level data and keep NA statewide data (e.g. in Illinois).